packages/styles/docs/sass.md

```diff
@@ -37,6 +37,11 @@ The `@carbon/styles` package provides the Sass files needed to include every
 style for the Carbon Design System. It includes entrypoints for themes, tokens,
 CSS helpers, components, and more.
 
+**Note: You may not need this package! If you're using `@carbon/react` you do
+not need to install `@carbon/styles`. Module paths in `@carbon/styles` and
+`@carbon/react` are synonymous - `@carbon/react` re-exports everything from
+`@carbon/styles`.**
```

Comment:
This might be overkill but would we want to add a line like 'For example `@carbon/styles/scss/components/button` is the same as `@carbon/react/scss/components/button`'